### PR TITLE
More CSS improvements

### PR DESCRIPTION
**Pull request by Dremora**
- Modularized a few more stylesheets
- Fixed issues related to upgrade of normalize.css
- Fixed avatar positioning on the ride page (broken by my previous PR)
- Now showing MasterCard icon
- Notifications don't block lower part of the screen anymore
- Restore broken loading spinner message

### CODE REVIEW

File: client/rides/components/RideRegistrationModal.jsx [hunk 1]
```diff
@@ -19,22 +19,22 @@ const PendingRegistration = () => (
...
 const CreateFundraiser = () => (
-  <div className="ride-registration--form">
+  <div>
     <p>
       You're all set! You've completed registration and we've received payment - all that's left
       to do now is to setup your fundraising page and train!
     </p>
     <SetupFundraising />
-    <CloseRideRegistrationModalButton className="btn-grey" text="Not right now" />
+    <CloseRideRegistrationModalButton text="Not right now" />
```
Comment (mwillmott) [inline client/rides/components/RideRegistrationModal.jsx]: As far as i can tell this is a breaking change. The whole reason for passing the `btn-grey` classname through i to make sure the button is the right colour.

**Comment by mwillmott** [comment]
OK, this is a pretty aggressive changing of some existing practice here. I'm all good with moving across to styled components but i feel there are a few things we're losing now. For instance, the purpose of having `spacing.css` is to make sure that we can start to standardise padding and margins across the site. If this doesn't happen then things get out of hand quickly and can start to look bad. Rather than moving away from modular css and just hardcoding spacing into styled components i'd like to see the same practice of standardising spacing stick with the change.

I appreciate that there is a lot of mess in the codebase which doesn't make use of the standardised spacing etc but i'd like to see us move towards that as we clean it up rather than away.

There are lots of changes here so i'm going to take some time to properly test it.

File: client/rides/components/RideRegistrationModal.jsx (hunk 1 above)
Comment (Dremora) [inline client/rides/components/RideRegistrationModal.jsx]: We don't have `btn-grey` class.

**Comment by Dremora** [comment]
I've worked on a [large-scale app](https://kayako.com) for two years, and I can tell you from my experience that spacing variables have introduced a lot more problems than they actually solved. In fact, pretty much all variables, mixins and any other pieces of reusable CSS did. Our codebase became a lot more maintainable once we removed those abstractions.
Padding and margin nearly always has to be fine-tuned. We've had plenty of places like `calc($padding-left - 1px)`.  Classes like `mt3` do not use user-friendly naming. Once you start adding global classes, you can't rely on component's own styles as a single source of truth.
When taking design from Sketch/Photoshop and converting it into CSS, you always have to measure distances (in pixels) and then convert them to variables/classes. A developer reading code has to do the reverse in order to find out what are the actual margins/paddings of an element. This abstraction is completely unnecessary.
See for yourself how much of these classes have actually been in use (answer: almost none). But it took me a lot of effort to figure out where some of them are coming from (because of SASS's magical style generation).

> If this doesn't happen then things get out of hand quickly and can start to look bad.
Well, I don't think inconsistent spacing is the main issue with our design :)

**Pull request merged**